po/da.po

```diff
@@ -14899,14 +14899,17 @@ msgstr "Officiel dokumentation"
 
 #: src/other-resources.md:8
 msgid "The Rust project hosts many resources. These cover Rust in general:"
-msgstr ""
+msgstr "Rustprojektet hoster mange resourcer. Disse dækker Rust generelt:"
```

Comment:
How about "Rust-projektet huser mange ressourcer."?

note: resourcer -> ressourcer. https://ordnet.dk/ddo/ordbog?query=ressource

Reply:
Yeah, that's much better!
```suggestion
msgstr "Rust-projektet huser mange ressourcer. Disse dækker Rust generelt:"
```